src/components/Card/Card.js

```diff
@@ -24,18 +24,14 @@ export default function QuestionCard({ questionDetails, height }) {
       default:
     }
   };
-  const getLabel = event => {
-    return {
-      __html: view === View.question ? question : answer,
-    };
-  };
+
   return (
     <Flippy flipOnClick={true} flipDirection='vertical' style={height}>
       <FrontSide onClick={clickHandler} className='front-side'>
-        <span dangerouslySetInnerHTML={getLabel()}></span>
+        <span>{question}</span>
```

Comment:
`dangerouslySetInnerHTML`  is to parse Html tags to be readable, text in list tag <li><ui> will display a list with bullet, removing this prop will display the Html tags.
mote about this can be found: [](https://reactjs.org/docs/dom-elements.html)

Currently, the card text that have Html tag display the tags, Please fix this by returning it back or different solution can work too :) 
Thanks!